packages/calcite-components/src/components/action-bar/action-bar.scss

```diff
@@ -53,5 +56,5 @@
 }
 
 .action-group--bottom {
-  @apply flex-grow justify-end;
```

Comment:
Why was this removed?

Reply:
With flex-grow, the `calcite-action-group` is filling up the space in full width mode. Any click on the space filled is `delegatingFocu`s in `calcite-action-group`. To avoid this, the empty space will be filled by `calcite-action-bar`  and `delegateFocus` will be removed from `calcite-action-bar`.